client/src/components/HeightClipper/HeightClipper.tsx

```diff
@@ -37,10 +40,37 @@ export class HeightClipper extends React.Component<
     };
     this.main = React.createRef();
     this.content = React.createRef();
+
+    this.debounced_mutation_callback = _.debounce(() => {
+      const height_clipper_node = ReactDOM.findDOMNode(this) as HTMLElement;
+      const untabbable_children_node = height_clipper_node.querySelector(
+        ".untabbable_children"
+      );
+
+      if (untabbable_children_node) {
+        this.setFocusableFalse(untabbable_children_node);
+      }
```

Comment:
Close! Not technically wrong, but I believe `findDOMNode` is considered shaky (looks like it's deprecated in React's strict mode, they'd probably fully deprecate it if it wouldn't be a huge breaking change). Refs are the preferred approach.

Set up a `this.untabbable_children = React.createRef();` in the constructor, set `ref={this.untabbable_children}` on the `.untabbable_children` node, and then you can check for/reference the corresponding DOM node directly with `this.untabbable_children.current`. This a few other places in this file that look to use some outdated practices (guessing they're what influenced this), please clean them up while you're in there.